frontend/src/pages/detail/components/Post/Post.styled.ts

```diff
@@ -0,0 +1,63 @@
+import styled from "@emotion/styled";
+import { Link } from "react-router";
+import { textOverflowEllipsis } from "@/styles/global.styled";
+
+export const PostBox = styled.li`
+  width: 100%;
+  height: 10rem;
+  border: 1px solid #bbbbbb;
+  background-color: #ffffffff;
+  margin-bottom: 1.25rem;
+  border-radius: 8px;
+  overflow: hidden;
+`;
+
+export const PostLink = styled(Link)`
```

Comment:
혹시 Link 컴포넌트 말고, a 태그의 `target="_blank"` 속성을 이용해서 새 탭으로 여는 건 어떨까요?
유저 이탈률을 최대한 줄이면 좋을 것 같아서요!ㅎㅎ

```suggestion
export const PostLink = styled.a`
```

Reply:
좋은 생각인 것 같습니다👍
외부로 열리는 기능이 좀 더 적합할 것 같은데 a태그를 사용해서 보안적으로 안전하고 브라우저나 스크린리더에 좀 더 친화적일 것 같습니다. 🙇‍♂️

[88c9922](https://github.com/woowacourse-teams/2025-moaon/pull/83/commits/88c99229a3a4ca908eee7eca36eb1a70b4fa2997)